applications/common/backend/src/main/resources/libs/kyanite/common/components/structurepage/head-schema.html

```diff
@@ -0,0 +1,29 @@
+<!--/*
+  ~
+  ~     Copyright (C) 2023 Dynamic Solutions
+  ~
+  ~     Licensed under the Apache License, Version 2.0 (the "License");
+  ~     you may not use this file except in compliance with the License.
+  ~     You may obtain a copy of the License at
+  ~
+  ~         http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~     Unless required by applicable law or agreed to in writing, software
+  ~     distributed under the License is distributed on an "AS IS" BASIS,
+  ~     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~     See the License for the specific language governing permissions and
+  ~     limitations under the License.
+  ~
+
+this breaks the page
+    <script type="text/javascript">
+      window.location.href = "${model.url}";
+    </script>
```

Comment:
i googled and people say, that it is good to have this script as a fallback, when the normal redirection doesn't work. However, applying that code makes the page blank and never load anything.
I added this comment just to start a discussion about this. @Mikilll94 @paweljozwicki

Reply:
After some discussions: we will provide some other solutions for redirects. For know it will be just a static page without any content